v2/api-validator/tests/server-tests/bad-request-body.test.ts

```diff
@@ -21,58 +21,60 @@ describe('Test request bodies missing one required property', () => {
 
   let accountId: string;
 
-  describe.each(postEndpoints)('$method $url', ({ operationId, url, schema }: EndpointSchema) => {
-    const [component] = schema.tags;
-    accountId = getCapableAccountId(component as keyof ApiComponents);
-
-    // Test multiple times due to the randomness of the generated payloads
-    describe.each([1, 2, 3, 4, 5, 6, 7, 8, 9, 10])('Attempt %d', () => {
-      const goodBody = JSONSchemaFaker.generate(schema.body as Schema);
-      if (!goodBody || typeof goodBody !== 'object') throw new Error('Unexpected body type');
-
-      const sendRequest = async (requestBody: JsonValue) => {
-        const operationFunction = client[schema.tags[0]]?.[operationId].bind(client);
-
-        try {
-          await operationFunction({ requestBody, accountId });
-        } catch (err) {
-          if (err instanceof ApiError) {
-            return err;
-          }
-          throw err;
-        }
-        throw new Error('Expected to throw');
-      };
-
-      for (const propertyPath of getPropertyPaths(goodBody)) {
-        describe(`Body without ${propertyPath.join('.')}`, () => {
-          let apiError: ApiError;
-
-          beforeAll(async () => {
-            const badBody = _.cloneDeep(goodBody);
-            deleteDeepProperty(badBody, propertyPath);
-
-            if (Object.hasOwn(badBody, 'idempotencyKey')) {
-              badBody['idempotencyKey'] = randomUUID();
+  if (postEndpoints.length > 0) {
+    describe.each(postEndpoints)('$method $url', ({ operationId, url, schema }: EndpointSchema) => {
```

Comment:
Use `describe.skipIf`. There are many examples in the code. You'll probably will have to do it on the top describe.